docs/resources/Message.md

```diff
@@ -345,6 +346,21 @@ There are multiple message types that have a `message_reference` object.
 - These messages have `message_id`, `channel_id`, and `guild_id`.
 - These messages will never have content, embeds, or attachments, mainly just the `message_reference` and `referenced_message` fields.
 
+###### Poll result messages
+
+- These are automatic messages sent after a poll has ended and the results have been finalized. (type 46)
+- These messages have `message_id` and `channel_id`, which point to the original poll message. The `channel_id` will be the same as that of the poll.
+- The author will be the same as the author of the poll, and they will mention the author of the poll.
+- These messages contain a `poll_result` embed that can have the following fields:
```

Comment:
The "**can** have the following fields" is confusing. Does this mean fields other than those saying "if applicable" are optional in poll result embeds?